### DISCUSSION TITLE
Is there support for `resolvable: false` for @key?

### DISCUSSION THREAD

**Discussion by cs-clarence** [async-graphql/async-graphql#1361] [Q&A]
I need to use [this](https://www.apollographql.com/docs/federation/entities#referencing-an-entity-without-contributing-fields) feature in federation. I haven't seen any docs or examples for the federation feature that uses this.

**Comment by akrantz01** [comment]
Would love to know if this is possible as well!

**Reply by akrantz01** [reply]
Looking at the code for exporting the SDL, it doesn't appear that this is possible at the moment. The only places `@key` is written to the schema is concerning the entity fields ([here](https://github.com/async-graphql/async-graphql/blob/40a781fd4660170b8e72b44bf4a1fe19724401e6/src/registry/export_sdl.rs#L370) and [here](https://github.com/async-graphql/async-graphql/blob/40a781fd4660170b8e72b44bf4a1fe19724401e6/src/registry/export_sdl.rs#L420)).

**Reply by akrantz01** [reply]
Opened a PR implementing this: #1378

**Reply by akrantz01** [reply]
It's been merged and is available in 6.0.7!
